lib/gnarails/cli/application.rb

```diff
@@ -7,175 +7,42 @@ class Application < Thor
 
       add_runtime_options!
 
-      WEBPACKS = %w[react vue angular elm stimulus].freeze
```

Comment:
These options are out of date, rendering some of the functionality a bit useless. 

I would recommend we adopt an approach similar to #154 , passing through rails args directly rather than, as Kevin put it, Playing cat-and-mouse with rails options (and potentially having to support multiple versions)